tests/foreman/cli/test_ping.py

```diff
@@ -12,57 +12,47 @@
 
 :TestType: Functional
 
-:CaseImportance: High
+:CaseImportance: Critical
 
 :Upstream: No
 """
 import pytest
 
 from robottelo import ssh
-from robottelo.test import CLITestCase
 
 
-class PingTestCase(CLITestCase):
-    """Tests related to the hammer ping command"""
+@pytest.mark.tier1
+@pytest.mark.upgrade
+def test_positive_ping(self):
+    """hammer ping return code
 
-    @pytest.mark.tier1
-    @pytest.mark.upgrade
-    def test_positive_ping(self):
-        """hammer ping return code
+    :id: dfa3ab4f-a64f-4a96-8c7f-d940df22b8bf
 
-        :id: dfa3ab4f-a64f-4a96-8c7f-d940df22b8bf
+    :steps:
+        1. Execute hammer ping
+        2. Check its return code, should be 0 if all services are ok else
+           != 0
 
-        :steps:
-            1. Execute hammer ping
-            2. Check its return code, should be 0 if all services are ok else
-               != 0
+    :expectedresults: hammer ping returns a right return code
+    """
+    result = ssh.command(f'hammer -u {self.foreman_user} -p {self.foreman_password} ping')
+    assert result.stderr == ''
```

Comment:
I wonder, how this assert worked for you here, for me
 ```
>       assert result.stderr == ''
E       AssertionError: assert b'' == ''
```

Reply:
This is missing a commit! `stderr` does need to be decoded to make this comparison, I just hadn't pushed the commit. Good catch!